### DISCUSSION TITLE
Tree view - table, card (etc.) records display in tree structure

### DISCUSSION THREAD

**Discussion by alcompstudio** [Budibase/budibase#9265] [Help]
Hi!

I just started learning the platform, very powerful functionality!
I want to know if it is possible to display a list of data (records from the database) in a tree structure? Like in this example
![image](https://user-images.githubusercontent.com/22731880/211090611-a82f65b9-6d5c-4f6e-a48a-3ae1bb7d7c5c.png)

Maybe in screens, tables, cards or other views, where it possible

That is, I want to make a relationship with another record in the same data table for one row of a record. And the second entry, in turn, will be related to the third entry, and so on.

Can this be done? In general, this is necessary purely for visualization, in order to build some kind of structural view (for example, a content structure for a future site).

**Comment by melohagan** [comment]
You can checkout using [custom components](https://docs.budibase.com/docs/custom-plugin) 

For example, @poirazis has added a [SuperTreeView](https://github.com/poirazis/bb-component-SuperTreeView) that you could take a look at.
Although for your use-case you may need to build something a bit more custom, like a tree view table.

**Reply by cpo1964** [reply]
poirazis is not really responsive ...
anybody out there who already used SuperTreeView and who is willing to share an budibase export with an example of usage ?

**Reply by mikesealey** [reply]
> poirazis is not really responsive

I think @poirazis is the most active Custom Component developer; You could try asking him on the [Budibase Discord Server](https://discord.com/invite/ZepTmGbtfF), alternatively you could ask on the GitHub repo for the SuperTreeView itself.

**Comment by alcompstudio** [comment]
Hi!

Thanks for the feedback! But I already found out for myself that this platform is not suitable for me, unfortunately ... This is more for internal use within the company, and I need to create something like a marketplace, with public user registration. And here there is no such thing. But anyway, thanks for the reply!
